### PR TITLE
feat: update home page

### PR DESCRIPTION
**Pull request by ChristanDaniel**
## Why

Update to the new homepage.

## Changes

- feat: create NewButton component
- feat: create TransparentCard component
- feat: create DaoAssetManagement component
- refactor: update SectionCard component in Home page
- refactor: update home page hero
- feat: create PartnersMarquee component

### CODE REVIEW

File: src/Icons/Arrow-right-circle.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import type { SvgProps } from '.'
```
Comment (guilherme-correa-s) [inline src/Icons/Arrow-right-circle.tsx:1]:
```suggestion
import type { SvgProps } from './'
```

File: src/components/CallToActionEndPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import Link from 'next/link'
+
+import NewButton from '../NewButton'
+
+import { ArrowRightCircle } from '@/Icons/Arrow-right-circle'
+
+import * as S from './styles'
+
+type ICallToActionEndPageProps = {
```
Comment (guilherme-correa-s) [inline src/components/CallToActionEndPage/index.tsx:9]:
```suggestion
type CallToActionEndPageProps = {
```

File: src/components/TransparentCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import * as S from './styles'
+
+import { ReactNode } from 'react'
+
+interface ITransparentCardProps {
```
Comment (guilherme-correa-s) [inline src/components/TransparentCard/index.tsx:5]: Interface or type of default props?

**Review by guilherme-correa-s — CHANGES_REQUESTED** [review]

**Review by Tiago138 — APPROVED** [review]

**Review by guilherme-correa-s — APPROVED** [review]
LGTM

**Pull request merged**